.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release artifact
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Restore Coursier cache
+      uses: coursier/cache-action@v3
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Release artifact
+      run: sbt release
```

Comment:
What confuses me is I don't understand where sbt comes from :-/ 

I usually use https://github.com/olafurpg/setup-scala which sets it up, but not sure which action brings it in here.

Reply:
IIRC the action to setup Java included it.

Reply:
Hah. Crazy. I think it's actually available on Linux by default: https://github.com/actions/virtual-environments/blob/51d02f26f42e53bb963bb6bc9754501b28a3d2ba/images/linux/scripts/installers/sbt.sh

Reply:
Hmm its in the Windows image installer scripts too, but not sure about the Mac one. 
We should have a matrix build to test on every platform as well as supported JDKs, right?

Reply:
That'd be overkill, tbh, builds might get crazy long - plus most of the stuff that can go wrong is in SBT's and effect system's realm, and they probably do have rich mbuild matrices.

JDKs on the other hand I don't think people are as vigilant.

Reply:
Build time should not be an issue as the builds run in parallel.

Having said that, I found that the macOS image does not come with SBT so I'll simplify it and just run on Ubuntu.